include/psp2/fpu.h

```diff
@@ -0,0 +1,147 @@
+/**
+ * \usergroup{SceFpu}
+ * \usage{psp2/fpu.h,SceFpu_stub}
+ */
```

Comment:
@Rinnegatamante already you said on the IRC, this module cannot use this lib for the hbl develop like other static libs.
can you add this information to here?